src/api/controllers/termsAndConditionsController.js

```diff
@@ -0,0 +1,55 @@
+import statusCode from '../../helpers/constants/status.codes';
+import models from '../models';
+
+const { TermsAndConditions } = models;
+
+/**
+ * @class TermsAndConditionController
+ * @description a class to get or update terms and conditions of Authors haven
+ */
+export default class TermsAndConditionController {
+  /**
+   *
+   * @param {Object} req - Request
+   * @param {Object} res  - Response
+   * @param {Function} next -Next
+   * @returns {Object} The response object
+   */
+  static async getTermsAndConditions(req, res) {
+    const termsDocument = await TermsAndConditions.findOne({
+      where: {
+        id: req.params.id
+      }
+    });
+    return res.status(statusCode.OK).json({
+      termsDocument
+    });
+  }
+
+  /**
+   *
+   * @param {Object} req - Request
+   * @param {Object} res  - Response
+   * @param {Function} next -Next
+   * @returns {Object} The response object
+   */
+  static async updateTermsAndConditions(req, res) {
+    const updatedTerms = req.body.termsAndConditions;
```

Comment:
how many terms and conditions do we have?? if there is only one I don't get the why of doing a findOne

Reply:
I am finding that specific document but there is no guarantee that we won't have any other document of terms and conditions in the future .that's exactly why  I am finding one